StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -1311,6 +1311,14 @@ public class STPPaymentHandler: NSObject {
             defer {
                 blockingDataTaskSemaphore.signal()
             }
+
+            // Check if the request failed due to attempting to redirect to an app universal link for Cash App
```

Comment:
Change:
```
// Check if the request failed due to attempting to redirect to an app universal link for Cash App
```

To reference "Custom Scheme URL" instead of Universal link... maybe something like:
```
// Check if the request failed due to attempting to redirect to a custom Scheme URL for Cash App
```